Documentation/local-kpm-registry.md

```diff
@@ -0,0 +1,68 @@
+## Running a local registry
+
+You may want to run a kpm registry locally, and then deploy your kpm registry to k8s from this "bootstrapping registry" if-you-will.
+
+It requires etcd to be present, so get up etcd first.
+
+```
+$ docker run --name tempetcd -dt -p 2379:2379 -p 2380:2380 quay.io/coreos/etcd:v3.0.6 /usr/local/bin/etcd -listen-client-urls http://0.0.0.0:2379,http://0.0.0.0:4001 -advertise-client-urls http://$127.0.0.1:2379,http://127.0.0.1:4001
```

Comment:
Thanks ant, great pointer on that. I went ahead and broke into two sections in another pushed commit:

* Using file system back-end
* Using default etcd back-end